docs/essentials/add-component-to-storybook.mdx

```diff
@@ -206,7 +206,7 @@ module.exports = {
   url: "https://demo.front-commerce.com",
   modules: ["./my-module"],
   serverModules: [
-    { name: "FrontCommerceCore", path: "./modules/front-commerce-core" },
+    { name: "FrontCommerce", path: "./modules/front-commerce" },
     { name: "Magento2", path: "./modules/magento2" },
```

Comment:
I wonder if this one should be 
```suggestion
    { name: "FrontCommerce", path: "server/modules/front-commerce" },
    { name: "Magento2", path: "server/modules/magento2" },
```
 instead?

Reply:
Totally!